Sources/Commands/UserToolchain.swift

```diff
@@ -25,10 +25,15 @@ struct UserToolchain: Toolchain {
     let sysroot: String?
 
 #if os(OSX)
+    var targetArgs: [String] {
+        return ["-target", "x86_64-apple-macosx10.10"]
+    }
+
     var platformArgs: [String] {
-        return ["-target", "x86_64-apple-macosx10.10", "-sdk", sysroot!]
+        return targetArgs + ["-sdk", sysroot!]
```

Comment:
Makes sense. Let's just use the canonical form of passing the arguments to Clang.